cpp/src/plasma/plasma.h

```diff
@@ -134,8 +134,9 @@ struct ObjectTableEntry {
   /// IPC GPU handle to share with clients.
   std::shared_ptr<CudaIpcMemHandle> ipc_handle;
 #endif
-  /// Set of clients currently using this object.
-  std::unordered_set<Client*> clients;
+  /// Reference count.
```

Comment:
Better documentation is: Number of clients currently using this object.

Reply:
Yes this is indeed better:)